resources/kcp/charts/kyma-environment-broker/templates/deployment.yaml

```diff
@@ -440,7 +441,7 @@ spec:
       - name: swagger-volume
         configMap:
           name: {{ include "kyma-env-broker.fullname" . }}-swagger
-      {{- if and (eq .Values.global.database.embedded.enabled false) (eq .Values.global.database.cloudsqlproxy.enabled true)}}
+      {{- if and (eq .Values.global.database.embedded.enabled false) (eq .Values.global.database.cloudsqlproxy.enabled true) (ne .Values.global.database.cloudsqlproxy.workloadIdentity.enabled false}}}
```

Comment:
In L420 you drop `cloudsql-instance-credentials` when `workloadIdentity.enabled = true`. Here, in L444, you include `cloudsql-instance-credentials` Volume when `workloadIdentity.enabled != false`. Let's think about the case when `workloadIdentity.enabled = false`, then:
```
name: cloudsql-proxy
image: {{ .Values.global.images.cloudsql_proxy_image }}
command: ["/cloud_sql_proxy",
          "-instances={{ .Values.global.database.managedGCP.instanceConnectionName }}=tcp:5432",
          "-credential_file=/secrets/cloudsql-instance-credentials/credentials.json"]
volumeMounts:
    - name: cloudsql-instance-credentials
      mountPath: /secrets/cloudsql-instance-credentials
      readOnly: true
```
and `cloudsql-instance-credentials` Volume won't be included.

Reply:
fixed